### ISSUE TITLE
Bug: `natural` sort ignores `order`

### ISSUE THREAD

**Issue by hugop95** [azat-io/eslint-plugin-perfectionist#405]
### Describe the bug

Please see the self-explicit code example below.

### Code example

Configuration:
```ts
{
  type: 'natural',
  order: 'desc'
}
```

Code:
```ts
interface Test {
    a
    b
}
```

No error is reported.

### ESLint version

8.57.0

### ESLint Plugin Perfectionist version

4.1.2

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Fixed in 82cb68a964fcb03869b415eea4d665ad305c8ca1.
Released in [v4.2.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v4.2.0).
